### ISSUE TITLE
Folder() instantiation fails with: AttributeError: 'parent' are invalid kwargs for this class

### ISSUE THREAD

**Issue by rajaramsrn** [ecederstrand/exchangelib#321]
Folder creation does not work as [documented](https://github.com/ecederstrand/exchangelib#folders). Fails with AttributeError for parent though parent is the required kwarg key for this class. My objective is to create a subfolder under inbox.  I am on exchangelib==1.10.6

```
from exchangelib import Account
from exchangelib import DELEGATE
from exchangelib import ServiceAccount
from exchangelib.folders import Folder

# Redacted username, password, etc.

credentials = ServiceAccount(username=domain + '\\' + username, password=password, max_wait=60)
account = Account(primary_smtp_address=username, credentials=credentials, autodiscover=True, access_type=DELEGATE)
f = Folder(parent=account.inbox, name="A Sub Folder")
```

Results in the following exception trace:

Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/Users/myname/exchpjt/lib/python2.7/site-packages/exchangelib/folders.py", line 98, in __init__
    super(Folder, self).__init__(**kwargs)
  File "/Users/myname/exchpjt/lib/python2.7/site-packages/exchangelib/properties.py", line 42, in __init__
    raise AttributeError("%s are invalid kwargs for this class" % ', '.join("'%s'" % k for k in kwargs.keys()))
AttributeError: 'parent' are invalid kwargs for this class

**Comment by ecederstrand** [comment]
This feature hasn't been released yet on PyPI. You'll have to install from GitHub for now. I'll get to it soon :-)

```bash
pip install git+https://github.com/ecederstrand/exchangelib.git@6a9ec3d1358a3d65fcb8eee310d1508a15ab04de
```

**Comment by rajaramsrn** [comment]
Ah, that explains! Will wait for the release. Thank you.

**Comment by ecederstrand** [comment]
I just released v1.10.7 which contains the new features.
